exist-core/src/test/java/xquery/xquery3/XQuery3Tests.java

```diff
@@ -26,7 +26,9 @@
 
 @RunWith(XSuite.class)
 @XSuite.XSuiteFiles({
-        "src/test/xquery/xquery3"
+        "src/test/xquery/xquery3",
+        "src/test/xquery/xquery3/transform",
+        //"src/test/xquery/xquery3/transform/fnTransform13a.xqm",
```

Comment:
should be removed or for good reasons?

Reply:
I wanted a record/reminder of how to run a single test. Edited comment to make that clear.